### ISSUE TITLE
Following the install instructions for docker-compose on aarch64 returns "Not found" - if aarch64 isn't supported, the documentation page should specify

### ISSUE THREAD

**Issue by sssheridan** [docker/docker.github.io#13538]
File: [compose/install.md](https://docs.docker.com/compose/install/)

I've installed Docker Engine on my Pi, and I came to this page to install docker-compose. The following cryptic error:

````
$ ./docker-compose --version
./docker-compose: line 1: Not: command not found
````
...arises because the file downloaded to `/usr/local/bin/docker-compose` (from `https://github.com/docker/compose/releases/download/1.29.2/docker-compose-Linux-aarch64`) is simply `Not found`.

If aarch64 isn't supported or its binaries aren't available, I would suggest one or more of the following:

- Specify on the documentation page "Linux (x86_64)" where it now reads "Linux";
- Add an option on the documentation page for Linux (aarch64), at least pointing to a URL where a user should go; 
- and/or place a file at https://github.com/docker/compose/releases/download/1.29.2/docker-compose-Linux-aarch64 with something like:
````
echo "aarch64 binaries not available. Please [install via pip/compile from source/whatever]: [URL a user should visit]"
````

**Comment by salt-lick** [comment]
Along these lines :

File: [compose/install.md](https://docs.docker.com/compose/install/)

The URL scheme specified for the 'Linux' section for composer versions  < 2 are not all available to download.

e.g.  `.../1.29.2/docker-compose-$(uname -s)-$(uname -m)` on a RPI would become
https://github.com/docker/compose/releases/download/1.29.2/docker-compose-Linux-aarch64 and that 404s 


For composer versions > 2: the URLs listed need to be updated to include the `v` next to the version number
e.g. 
| URL  | HTTP CODE |
|---|---|
| `.../2.0.1/docker-compose-$(uname -s)-$(uname -m)` | 404 |
| `.../v2.0.1/docker-compose-$(uname -s)-$(uname -m)` | 200 |

**Comment by danBamikiya** [comment]
@salt-lick

>For composer versions > 2: the URLs listed need to be updated to include the v next to the version number

There's an entire page for installing versions > 2:
![docker_docs_website_issue_2](https://user-images.githubusercontent.com/58262528/137597919-caff84f3-9dd9-4306-a053-18be18d95818.png)


>The URL scheme specified for the 'Linux' section for composer versions < 2 are not all available to download.

@salt-lick @sssheridan From this comment: https://github.com/docker/compose/issues/8758#issuecomment-936444604 the maintainers suggest replacing `$(uname -m)` with `aarch64`

@thaJeztah do you think anything can be done about this?

**Comment by salt-lick** [comment]
> @salt-lick
> 
> > For composer versions > 2: the URLs listed need to be updated to include the v next to the version number
> 
> There's an entire page for installing versions > 2: 

_Forest through the trees...._ but the blue on slightly bluer is a bit harder to read. Thanks for pointing that out.

**Comment by sssheridan** [comment]
> @sssheridan Hi the instructions on that page has been updated. Does it work for you now?

I returned to this project now, and installed docker compose through apt without issue. So this issue has been resolved.

In case anyone arrives here from Google still having an issue, try `apt install docker-compose` instead of `docker-compose-plugin`. On my Raspberry Pi 4 running Ubuntu 220.4, I have the apt packages `docker` and `docker-compose` installed. I'm a little confused about the `docker-compose-plugin` in the docs, and might open another issue on that topic, but the issue in this thread is closed, for me at least.
